apis/Google.Cloud.Language.V1Beta1/Google.Cloud.Language.V1Beta1.Tests/DocumentTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using Xunit;
+
+namespace Google.Cloud.Language.V1Beta1.Tests
+{
+    public class DocumentTest
+    {
+        [Fact]
+        public void FromHtml()
+        {
+            string text = "<b>Foo</b>";
+            var doc = Document.FromHtml(text, "en");
+            Assert.Equal(new Document { Content = text, Language = "en", Type = Document.Types.Type.Html }, doc);
+            doc = Document.FromHtml(text);
+            Assert.Equal(new Document { Content = text, Type = Document.Types.Type.Html }, doc);
+        }
+
+        [Fact]
+        public void FromText()
+        {
+            string text = "Foo";
+            var doc = Document.FromPlainText(text, "en");
+            Assert.Equal(new Document { Content = text, Language = "en", Type = Document.Types.Type.PlainText }, doc);
+            doc = Document.FromHtml(text);
```

Comment:
Doh doh doh! Need to adjust the method name, too. (And of course, Travis failed correctly...)